update_formspec.lua

```diff
@@ -227,10 +249,43 @@ function travelnet.update_formspec(pos, puncher_name, fields)
 			S("move up"),
 			S("move down")
 		)
+	if paging then
+		if page_number > 2 then
+			formspec = formspec .. ("button[0,9.2;2,1;first_page;%s]"):format(minetest.formspec_escape(S("<<")))
+		end
+		if page_number > 1 then
+			formspec = formspec .. ("button[2,9.2;2,1;prev_page;%s]"):format(minetest.formspec_escape(S("<")))
+		end
+		formspec = formspec
+			.. ("label[5,9.4;%s]"):format(minetest.formspec_escape(S("Page @1/@2", page_number, pages)))
+			.. ("field[20,20;0.1,0.1;page_number;Page;%i]"):format(page_number)
+			.. ("field[20,20;0.1,0.1;pos2str;Pos;%s]"):format(minetest.pos_to_string(pos))
```

Comment:
I don't like this usage of hidden fields, I can crash the server by changing the `pos2str` field.

Reply:
@luk3yx will take care of that when I get to this one https://github.com/mt-mods/travelnet/issues/33

I'm pretty certain the `page_number` one I added is okay though, worst you can do is skip to a page without clicking as many times, which should not be an issue, I considered using a context var but I'm not sure I can in this case, will take another pass at it when I do #33, but it's technically just visual data anyways so it kinda belongs in the form, a little conflicted TBH